config.go

```diff
@@ -145,6 +146,7 @@ type config struct {
 	Proxy                string        `long:"proxy" description:"Connect via SOCKS5 proxy (eg. 127.0.0.1:9050)"`
 	ProxyPass            string        `long:"proxypass" default-mask:"-" description:"Password for proxy server"`
 	ProxyUser            string        `long:"proxyuser" description:"Username for proxy server"`
+	Prune                uint32        `long:"prune" description:"Prune already validated blocks from the database. Must specify a target size in MiB (minimum value of 1024)"`
```

Comment:
Description doesn't match chosen min size constant value.

Reply:
Addressed in the latest push